### PR TITLE
Add category-ir

### PR DESCRIPTION
**Pull request by molaeiali**
Iranian websites that will not open without an Iranian IP

### CODE REVIEW

**Comment by rootmelo92118** [comment]
@IceCodeNew

**Comment by molaeiali** [comment]
> Please remove attributes.

Done

File: data/category-ir-bank [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Iranian bank websites which will not open without an Iranian IP
+# Use to make them direct connections so they open inside Iran
+
+cbi.ir
```
Comment (IceCodeNew) [inline data/category-ir-bank:4]: pls sort these domains

File: data/category-ir-bourse [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Iranian stock exchange websites which will not open without an Iranian IP
+# Use to make them direct connections so they open inside Iran
```
Comment (IceCodeNew) [inline data/category-ir-bourse:2]: this comment does not necessarily have to be introduced in every sub-category.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Review by IceCodeNew — APPROVED** [review]

File: data/category-ir [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Iranian websites which will not open without an Iranian IP
+# Use to make them direct connections so they open inside Iran
+
```
[the remark this answers is not in the document]
Comment (IceCodeNew) [reply data/category-ir:3]: please delete all of the comments similar to those shown here.

File: data/category-ir-bank (hunk 1 above)
Comment (molaeiali) [reply data/category-ir-bank]: Done

File: data/category-ir-bourse (hunk 2 above)
Comment (molaeiali) [reply data/category-ir-bourse]: Removed

File: data/category-ir (hunk 3 above)
[the remark this answers is not in the document]
Comment (molaeiali) [reply data/category-ir:3]: Done

**Comment by Loyalsoldier** [comment]
I recommend change the category name to pattern `category-${category_name}-${country_iso_code}`

**Comment by molaeiali** [comment]
> I recommend change the category name to pattern `category-${category_name}-${country_iso_code}`

There is already some categories with `category-${country_iso_code}-${category_name}` format in the project, like: category-ru-gov

I think differentiating by country is more beneficial for the people living in those countries than by category, 

But I'll do what you want if you insist on it.

**Comment by Loyalsoldier** [comment]
@molaeiali  I have deleted my recommendation.

**Review by IceCodeNew — APPROVED** [review]

**Comment by EryX0** [comment]
why not just include all Iranian domains ? @molaeiali
so all Iranian domains would be resolved with domestic ip and dns ( dns routing ).

**Comment by molaeiali** [comment]
> why not just include all Iranian domains instead of only the ones that need Iranian ip ? @molaeiali
> so all Iranian domains would be resolved with domestic ip and dns ( dns routing ).

Not necessarily, a server can be in Iran but blocked by government

**Comment by EryX0** [comment]
> Not necessarily, a server can be in Iran but blocked by government

would you give an example for this? a site thats registered on [this database](https://eservices.ito.gov.ir/page/iplist#) and is blocked by the government ?
its a more efficent way to browse with dns and ip routings making direct connections to domestic websites domains and ips.

**Pull request merged**